Explorer/Assets/DCL/FeatureFlags/FeatureFlagsProviderExtensions.cs

```diff
@@ -1,56 +1,37 @@
-﻿using System;
-using System.Threading;
+﻿using System.Threading;
 using CommunicationData.URLHelpers;
 using Cysharp.Threading.Tasks;
 using DCL.Web3;
+using System.Collections.Generic;
 
 namespace DCL.FeatureFlags
 {
     public static class FeatureFlagsProviderExtensions
     {
-        private const string ARG_URL = "--feature-flags-url";
-        private const string ARG_HOSTNAME = "--feature-flags-hostname";
+        private const string ARG_URL = "feature-flags-url";
+        private const string ARG_HOSTNAME = "feature-flags-hostname";
 
         public static async UniTask<FeatureFlagsConfiguration> InitializeAsync(
             this IFeatureFlagsProvider featureFlagsProvider,
             Web3Address? userAddress,
+            Dictionary<string, string> appParameters,
             CancellationToken ct)
         {
             FeatureFlagOptions options = FeatureFlagOptions.ORG;
-            GetOptionsFromProgramArgs(out URLDomain? programArgsUrl, out string? programArgsHostName);
 
-            if (programArgsUrl != null)
-                options.URL = programArgsUrl.Value;
+            // App parameters example:
+            // #!/bin/bash
+            // ./Decentraland.app --feature-flags-url https://feature-flags.decentraland.zone --feature-flags-hostname localhost
 
-            if (programArgsHostName != null)
-                options.Hostname = programArgsHostName;
+            if(appParameters.ContainsKey(ARG_URL))
```

Comment:
You could do:
```
if (appParameters.TryGetValue(ARG_URL, out string hostname))
    options.Hostname = hostname;
```
Same for other params

Reply:
Done at https://github.com/decentraland/unity-explorer/pull/1511/commits/d4dea5ea554d5c85810fba7a9f38021d0ae096c5